builders/manage/dapp/manage-tokens.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Manage Tokens
+description: Learn how the appchain governor can manage the token of a Tanssi-powered appchain using the dApp to mint them, configure the EIP-1559 fee market, and more.
+---
+
+# Manage Tokens
+
+## Introduction {: #introduction }
+
+Any appchain deployed through Tanssi is sovereign and free to define the governance model that best fits its use case. The appchain governor has superpowers over the chain administration, meaning privileged functions, such as updating the runtime, fall within their reach.
+
+There are some actions related to the native token management that are available to the appchain governor on the Tanssi dApp:
+
+- **Mint tokens** - mints new tokens, increasing the total supply
+- **Update balances** - increases or decreases the balance of an account, affecting the total supply
+- **Transfer tokens** - executes a forced token transfer from one account to another
+- **Configure gas dynamics** - only available on EVM-compatible appchains, this action changes the [EIP-1559](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1559.md){target=\_blank} configuration, affecting the transaction pricing mechanism
+
+In this guide, you'll learn how to execute the previously listed actions using the [Tanssi dApp](https://apps.tanssi.network){target=\_blank}.
+
+## Checking Prerequisites {: #checking-prerequisites }
+
+For the examples in this guide, you will need to have the following:
+
+- A Tanssi appchain (Snap or Dedicated)
```

Comment:
As this becomes repetitive, we should move this to a text snippet. @eshaben wdyt?

Reply:
Yes! Already created a ticket to address this:
https://opslayer.atlassian.net/jira/software/c/projects/DT/boards/108?selectedIssue=DT-213